packages/ui-file-drop/src/FileDrop/index.js

```diff
@@ -67,14 +67,14 @@ const IS_MS = isBrowserMS()
 category: components
 ---
 **/
+@withStyle(generateStyle)
 @deprecated('8.0.0', {
   label: 'renderLabel',
   enablePreview: 'shouldEnablePreview',
   allowRepeatFileSelection: 'shouldAllowRepeats',
   allowMultiple: 'shouldAllowMultiple'
 })
```

Comment:
We should remove these props from the component since they are deprecated.

Reply:
yep, I'm planning to do the removals in a later commit